src/main/java/de/qabel/desktop/ui/accounting/identitycontextmenu/IdentityContextMenuController.java

```diff
@@ -0,0 +1,135 @@
+package de.qabel.desktop.ui.accounting.identitycontextmenu;
+
+import de.qabel.core.config.Identity;
+
+import de.qabel.core.config.IdentityObserver;
+import de.qabel.core.repository.IdentityRepository;
+import de.qabel.core.repository.exception.PersistenceException;
+import de.qabel.desktop.config.ClientConfig;
+
+import de.qabel.desktop.ui.AbstractController;
+import de.qabel.desktop.ui.accounting.qrcode.QRCodeController;
+import de.qabel.desktop.ui.accounting.qrcode.QRCodeView;
+import javafx.application.Platform;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Label;
+import javafx.scene.control.TextInputDialog;
+import javafx.scene.input.MouseEvent;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Pane;
+import javafx.scene.layout.VBox;
+import org.controlsfx.control.PopOver;
+
+import javax.inject.Inject;
+import java.net.URL;
+import java.util.*;
+
+public class IdentityContextMenuController extends AbstractController implements Initializable {
+
+    ResourceBundle resourceBundle;
+
+    @Inject
+    private Identity identity;
+
+    @Inject
+    private ClientConfig clientConfiguration;
+
+    @Inject
+    private Pane layoutWindow;
+
+    @FXML
+    private AnchorPane menuQR;
+
+    @FXML
+    private VBox vboxMenu;
+
+    @FXML
+    private Pane avatarContainer;
+
+    @Inject
+    private IdentityRepository identityRepository;
+
+    public TextInputDialog dialog;
+
+    private QRCodeView qrcodeView;
+    private QRCodeController qrcodeController;
+    private PopOver popOver;
+    private Label alias;
+
+    @Override
+    public void initialize(URL location, ResourceBundle resources) {
+        resourceBundle = resources;
+
+        identity.attach(new IdentityObserver() {
+            @Override
+            public void update() {
+                if (alias != null) {
```

Comment:
I think this is dead code. The alias is never used so nobody is interested in an update